app/src/main/java/de/danoeh/antennapod/ui/screen/playback/SleepTimerDialog.java

```diff
@@ -169,11 +288,156 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         return builder.create();
     }
 
+    private void confirmAlwaysSleepTimerDialog() {
+        @SuppressLint("VisibleForTests")
+        AlertDialog dialog = new MaterialAlertDialogBuilder(requireContext())
+                .setTitle(R.string.sleep_timer_without_continuous_playback_title)
+                .setMessage(R.string.sleep_timer_without_continuous_playback_message)
+                .setNegativeButton(R.string.sleep_timer_without_continuous_playback_disable_playback,
+                        (dialogInterface, i) -> {
+                            // disable continuous playback and also disable the auto sleep timer
+                            UserPreferences.setFollowQueue(false);
+                            chAutoEnable.setChecked(false);
+                            refreshUiState();
+                        })
+                .setPositiveButton(R.string.sleep_timer_without_continuous_playback_proceed_with_timer,
+                        (dialogInterface, i) -> {
+                            chAutoEnable.setChecked(true);
+                        })
+                .create();
+
+        // don't let the user cancel this without selecting something
+        dialog.setCancelable(false);
+        dialog.setCanceledOnTouchOutside(false);
+
+        dialog.show();
+        // mark the disable continuous playback option in red
+        dialog.getButton(androidx.appcompat.app.AlertDialog.BUTTON_NEGATIVE)
+                .setTextColor(ContextCompat.getColor(requireContext(), android.R.color.holo_red_dark));
```

Comment:
We have our own error color somewhere if I remember correctly. I think that color even adapts to the theme. I think `ThemeUtils.getColorFromAttr` or something like that should help